app/views/Project/EntryEdit/LeftPane/SimplifiedTextView/index.tsx

```diff
@@ -200,9 +201,12 @@ function SimplifiedTextView(props: Props) {
         setTextToAssist(newTextToAssist);
     }, [resetTextSelection]);
 
-    const handleAssistedEntryAdd = useCallback((newEntry: EntryInput) => {
+    const handleAssistedEntryAdd = useCallback((
+        newEntry: EntryInput,
+        geoAreaOptions?: GeoArea[],
+    ) => {
         if (onAssistedEntryAdd) {
-            onAssistedEntryAdd(newEntry);
+            onAssistedEntryAdd(newEntry, geoAreaOptions);
         }
         setTextToAssist(undefined);
     }, [onAssistedEntryAdd]);
```

Comment:
Is there a reason we're not adding setTextToAssist in the dependency list?

Reply:
Not necessary if it is a setter function from `React.useState`